src/models/activityLog.model.ts

```diff
@@ -5,6 +5,7 @@ import mongoose, { Schema, Document } from 'mongoose';
 export interface ActivityLog extends Document {
   kind: 'ActivityLog';
   userId: mongoose.Types.ObjectId;
+  applicationId: mongoose.Types.ObjectId;
```

Comment:
@TaiKamilla 

I believe the logic is working, but please put the applicationId under metadata, so same level than the url for example

Reply:
Do you want me to remove it from here, or just add it to the metadata too?

Reply:
please remove it